lib/containers/common.pm

```diff
@@ -86,8 +86,10 @@ sub install_docker_when_needed {
                 # docker package can be installed
                 zypper_call('in docker', timeout => 300);
 
-                # Restart firewalld if enabled before. Ensure docker can properly interact
-                systemctl 'try-restart firewalld';
+                # Restart firewalld if enabled before. Ensure docker can properly interact (boo#1196801)
+                if (script_run('systemctl is-active firewalld') == 0) {
```

Comment:
This will not work if firewalld does not exist, rather check if firewalld is present.

Reply:
Why not? It's a `script_run`, not `assert_script_run`.
See also https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/services/sshd.pm#L55

Reply:
True, it will work.
It's just weird, it is failing now because there is no firewalld. https://openqa.suse.de/tests/8325047#step/docker_runc/180